spec/hbw/features/file_upload_spec.rb

```diff
@@ -72,5 +72,23 @@
       expect(preview_row).to have_no_text 'logo.svg'
       expect(preview_row).to have_no_css  "img[alt='logo.svg']"
     end
+
+    scenario 'should attach to second file_upload' do
+      attach_files('homsOrderDataUploadedFileAnother', files_to_attach)
+
+      preview_another = page.all('.files-preview-row')[0]
+      form_group = page.all('.form-group')[0]
+      file_input = form_group.find_field('homsOrderDataUploadedFileAnother', type: :file, visible: :hidden)
+
+      expect(file_input['name']).to  eq('homsOrderDataUploadedFileAnother')
```

Comment:
You have found this field by name, so you're testing Capybara now. Don't do that)